analysis/plotting.py

```diff
@@ -141,10 +142,18 @@ def coverage_growth_plot(self, benchmark_df, axes=None):
         axes.set(ylabel='Edge coverage')
         axes.set(xlabel='Time (hour:minute)')
 
-        ticks = np.arange(
-            0,
-            snapshot_time + 1,  # Include tick at end time.
-            snapshot_time / _DEFAULT_TICKS_COUNT)
+        if self._logscale:
+            axes.set_xscale('log')
+            ticks = np.logspace(
+                np.log10(5 * 60),  # Start from minute 5.
+                np.log10(snapshot_time + 1),  # Include tick at end time.
+                _DEFAULT_TICKS_COUNT)
+        else:
+            ticks = np.arange(
+                0,
```

Comment:
Does this need to match the logscale plot ? Should this start from snapshot time as well, will graph look weird [it currently does when we look at results in realtime, e.g. after 30 minutes, it shows empty ticks, values at like 5 min, 10 min, any tick < 15 min(first snapshot)..

Reply:
Yes, it makes sense to match them.